### PR TITLE
S3.2.2 one liner launch

### PR DESCRIPTION
**Pull request by RamiAwar**
# One-liner launch command

- Added Dockerfile
- Added supervisord config to launch BE & FE as processes on one container
- Added docker run command that includes volume mounting for persisting the database across container runs/updates
- Uploaded image to Dockerhub and updated README with launch directions

### CODE REVIEW

File: text2sql-backend/pyproject.toml [hunk 1]
```diff
@@ -11,21 +11,15 @@ python = ">=3.11,<3.12"
 fastapi = "0.100.0"
 uvicorn = {extras = ["standard"], version = "^0.21.1"}
 llama-index = "^0.9.42.post2"
-sqlparse = "^0.4.3"
-psycopg2 = "^2.9.6"
+psycopg2-binary = "^2.9.9"
```
Comment (anthony2261) [inline text2sql-backend/pyproject.toml:14]: Are you sure using the binary version here is ok? Don't personally know what the difference is, but I'd rather stick to regular psycopg2

File: text2sql-backend/dataline/config.py
Comment (anthony2261) [inline text2sql-backend/dataline/config.py:1]: Just curious, what issue came up that required this fix?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+COPY text2sql-backend/dataline ./dataline
+COPY text2sql-backend/alembic ./alembic
+COPY text2sql-backend/alembic.ini .
+
+WORKDIR /home/dataline
+
+RUN mkdir -p /home/.dataline
+
+# Supervisord will forward the env vars to the subprocess envs
+ENV SQLITE_PATH="/home/.dataline/db.sqlite3"
```
Comment (anthony2261) [inline Dockerfile:91]: I don't see this being used anywhere, and in the config we're manually specifying the path

File: text2sql-backend/pyproject.toml (hunk 1 above)
Comment (RamiAwar) [reply text2sql-backend/pyproject.toml:14]: It should be fine, it's just easier to install as a binary, less headachy.

Author is the same https://pypi.org/user/piro/

File: text2sql-backend/dataline/config.py
Comment (RamiAwar) [reply text2sql-backend/dataline/config.py:1]: Don't remember :D

File: Dockerfile (hunk 2 above)
Comment (RamiAwar) [reply Dockerfile:91]: In the config we specify a default. But this overrides it.

File: text2sql-backend/dataline/config.py
Comment (RamiAwar) [reply text2sql-backend/dataline/config.py:1]: It wasn't configured properly. I want SQLITE_DSN to be dynamic based on SQLITE_PATH, so we need a root validator to do this.

And for sqlite path, I wanted the default location to be inside dataline/backend/dataline/configuration, where configuration would be an empty folder that we'd mount a volume into.

File: Dockerfile (hunk 2 above)
Comment (anthony2261) [reply Dockerfile:91]: This is what I was looking for 👍 https://docs.pydantic.dev/latest/concepts/pydantic_settings/#case-sensitivity

**Pull request merged**